biolink-model.yaml

```diff
@@ -3317,6 +3343,14 @@ classes:
   gene:
     is_a: gene or gene product
     aliases: ['locus']
+    slots:
+      - id
+      - name
+      - synonym
+      - xrefs
+    slot_usage:
+      - name:
+          description: Gene symbol
```

Comment:
this is terminologically awkward as the full name (e.g. "sonic hedgehog") is sometimes referred to as 'name'

Reply:
we'll also want description